### PR TITLE
936 video

### PR DESCRIPTION
**Pull request by Rowansdabomb**
## What problem does this PR solve?
#936  Add Supports for Video article

## How did you fix the problem?

Changed `Featured Image` tab to be `Featured Media` that allows user to choose one of the two

image

Shout out to Ron for laying out the ground work for this 
Shout out to Atsushi for his shout out to Ron

### CODE REVIEW

**Comment by jumbosushi** [comment]
Thanks a lot for the help! What do you think about breaking apart the video migration file to separate migrations? I noticed that we have missed migrations for Articles prior to this PR, and it's being included in this video migration. 

Maybe to `0023_publishable_constraints_default` and `0024_video_meta_fields`

File: dispatch/modules/content/models.py [hunk 1]
```diff
@@ -451,10 +451,33 @@ def get_absolute_url(self):
...
+    authors = ManyToManyField(Author, related_name='video_authors')
+    tags = ManyToManyField('Tag')
+
+    created_at = DateTimeField(auto_now_add=True)
+    updated_at = DateTimeField(auto_now=True)
+
+    AuthorModel = Author
+
+    # Overriding
+    def save(self, **kwargs):
```
Comment (jumbosushi) [inline dispatch/modules/content/models.py]: We aren't doing anything special in `save()` (yet) so we can remove this method

Comment (Rowansdabomb) [reply dispatch/modules/content/models.py]: Good catch! I forgot I removed the custom save operations

**Comment by Rowansdabomb** [comment]
I like the idea of separating the migrations

**Comment by jumbosushi** [comment]
We should add columns too for `/videos/`

image

**Comment by Rowansdabomb** [comment]
I should have already added a preview, author, created_at, and updated_at
column. Here is a picture of how it looks on my machine.

[image: image.png]

*Rowan Skye*

**Pull request merged**